actions/email.go

```diff
@@ -89,6 +95,58 @@ func (email *EmailAction) Send(content map[string]string) error {
 	return nil
 }
 
+func (email EmailAction) sendEmailWithCustomClient(addr string, a smtp.Auth, from string, to []string, msg []byte) error {
```

Comment:
maybe add some verbosity to this function? var names, logs, errors and docstrings(not necessary), to look feel and stay readable like the previous sendEmail implementation.

Reply:
yes, sure. I've added a comment. will think over it more.